### PR TITLE
fixed connect() to call xmlrpc.createSecureClient() for  https connections if secure flag is set

### PR DESCRIPTION
**Pull request by oghmjos**
…tion if secure flag was set

### CODE REVIEW

File: src/testlink.ts [hunk 1]
```diff
@@ -72,7 +72,10 @@ export default class TestLink {
      * Establish connection
      */
     connect(): void {
-        this.rpcClient = xmlrpc.createClient({host: this.host, port: this.port, path: this.rpcPath});
+        let options = { host: this.host, port: this.port, path: this.rpcPath };
```
Comment (Nyaran) [inline src/testlink.ts:75]: As `options` is unmutable here, `const` is a better choice.
```suggestion
        const options = { host: this.host, port: this.port, path: this.rpcPath };
```

**Review by Nyaran — CHANGES_REQUESTED** [review]

**Comment by Nyaran** [comment]
Many thanks for your pull request, address the change requested and I'll integrete it :)

File: src/testlink.ts (hunk 1 above)
Comment (oghmjos) [reply src/testlink.ts]: sure, that's true

**Comment by oghmjos** [comment]
> Many thanks for your pull request, address the change requested and I'll integrete it :)

Done. It would be nice if you issue npm release 2.0.3 with this bug fix, so that we would not need to deal with private repo/cache with bugfix npm pkg.

**Comment by Nyaran** [comment]
> > Many thanks for your pull request, address the change requested and I'll integrete it :)
> 
> Done. It would be nice if you issue npm release 2.0.3 with this bug fix, so that we would not need to deal with private repo/cache with bugfix npm pkg.

Sure, I will publish the new version today or tomorrow at the latest

**Review by Nyaran — APPROVED** [review]

**Pull request merged**